src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  text-align: center;
+  line-height: 300px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  flex-basis: 300px;
+  flex-shrink: 0;
+  flex-grow: 1;
+}
+
+.b1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.b2 {
+  background-color: rgb(212.5, 0, 0);
+}
+
+.b3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.b4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.b5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.b6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
```

Comment:
you can write the initial order by default, and in media just change the ones that need to be changed. This way you will need to write much less code

Reply:
Ahh so you only need to specify the order for the exact ones that need to be shifted? Not all the blocks you are trying to move? ie. Once you specify the order of some blocks the rest file in according to the order set by the rules placed on the individual blocks?